double_entry.gemspec

```diff
@@ -17,6 +17,32 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry:
+ - New table `double_entry_line_metadata` has been introduced and is required for
+   aggregate reporting filtering. Existing applications will have to manually
```

Comment:
Perhaps tell them that if the table doesn't exist, aggregation will break? "and is required or aggregate reporting filtering will not work" or something like that? Too harsh?